libs/judicial-system/types/src/lib/user.ts

```diff
@@ -32,7 +32,7 @@ export interface User {
   loginCount?: number
 }
 
-interface InstitutionUser {
+export interface InstitutionUser {
```

Comment:
Please consider not exporting.  The interface is not being used outside this file.